doc/config.md

```diff
@@ -554,6 +554,7 @@ Default: 1
 
 Increase verbosity.  Mirrors the "-v" switch on the command line.
 For example, using "-v -v" on the command line is the same as `verbose=2`.
+2 is the highest currently-supported verbosity.
```

Comment:
```suggestion
3 is the highest currently-supported verbosity.
```

Reply:
Thank you for the heads-up! Fixed.

Reply:
2 is currently filling up my disks. I hope that I do not need 3! 😆